### ISSUE TITLE
issues on reopening minimized window

### ISSUE THREAD

**Issue by Iodid** [ppy/osu#1473]
2 seperate issues that may happen on reopening a minimized lazer window:

1. **black screen on bringing lazer window back up; highly reproduceable**
The music is still playing, but you can't directly interact with the (now vanished) window content with your mouse anymore. Keyboard input doesn't work as well.
When you try to exit lazer using alt+F4 or with the 'x' on the window border the "see you next time" jingle plays, but the window doesn't close. The window closes only after pressing alt+F4 or the 'x' a second time.
On the black screen the fps rises to 6000+ from ~1500 during the regular working main menu.
The issue is highly reproduceable for me and is not limited to the main menu.

2. **main menu background and menu elements off center after bringing the window back up**
On reopening a minimized laazer window the main menu background and elements center on the mouse , even if it is far outside the window border. On big enough screen resolutions / small enough lazer windows the main menu background and menu elements may become fully inaccessible due to them centering too far outside the window.
This is probably a low priority / low impact bug due to the main menu recentering when you bring back the cursor inside the window.

**Comment by peppy** [comment]
Probably the same thing reported here: https://github.com/ppy/osu/issues/1332.

**Comment by Iodid** [comment]
Another small note: Those problems seem to be windowed mode specific. I couldn't reproduce them with either borderless-window or fullscreen yet.
